jobs/storage_test.groovy

```diff
@@ -0,0 +1,57 @@
+evaluate(new File("${WORKSPACE}/common.groovy"))
+
+name = "storage_backend"
+repoName = 'charts'
+
+job(name) {
+  description """
+    <p>Runs the <a href="https://github.com/deis/workflow-e2e">e2e tests</a> against a <a href="https://github.com/deis/charts/tree/master/${defaults.workflowChart}">${defaults.workflowChart}</a> chart</p>
+  """.stripIndent().trim()
+
+  scm {
+    git {
+      remote {
+        github("deis/${repoName}")
+      }
+      branch('master')
+    }
+  }
+
+  logRotator {
+    daysToKeep defaults.daysToKeep
+  }
+
+  triggers {
+    cron('@daily')
+  }
+
+  wrappers {
+    timeout {
+      absolute(30)
+      failBuild()
+    }
+    timestamps()
+    colorizeOutput 'xterm'
+    credentialsBinding {
+      string("GCLOUD_CREDENTIALS", "246d6550-569b-4925-8cda-e11a4f0d6803")
+      string("GCS_KEY_JSON", "71dd890b-e4ce-4483-97ca-a7c286c2381c")
+    }
+  }
+
+  steps {
+    shell '''
+      #!/usr/bin/env bash
+
+      set -eo pipefail
+
+      export STORAGE_TYPE="gcs"
```

Comment:
It could be handy to leave this a generic `storage_backend_e2e` job and expose `STORAGE_TYPE` as a parameter that can be supplied when the job is run (defaulting to `gcs` or some other default if not supplied).  then, this same job dsl/config could be used to run against all needed storage backends...

If this might be a good idea, we can create a ticket to work on at a later date; not needed for this PR, just a thought.